presto-main/src/main/java/com/facebook/presto/testing/LocalQueryRunner.java

```diff
@@ -504,7 +504,7 @@ private MaterializedResult executeInternal(Session session, @Language("SQL") Str
             TaskContext taskContext = createTaskContext(executor, session);
 
             List<Driver> drivers = createDrivers(session, sql, outputFactory, taskContext);
-            drivers.stream().map(closer::register);
+            drivers.forEach(closer::register);
```

Comment:
IntelliJ warns that "Result of 'Stream.map()' is ignored". It is the "Result of method call ignored" inspection and is enabled by default.